src/styles/globals.css

```diff
@@ -0,0 +1,97 @@
+/* Global Styles */
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
+@import url("https://pro.fontawesome.com/releases/v5.10.0/css/all.css");
+
+:root {
+  --miracle-color-yellow: #ffd369;
+  --miracle-color-blue: #73d2de;
+  --miracle-color-red: #fc474d;
+  --miracle-color-green: #6fcf97;
+  --miracle-color-purple: #8361c7;
+  --miracle-color-light: #ffffff;
+  --miracle-color-medium: rgb(181, 186, 194);
+  --miracle-color-dark: #222831;
+}
+
+:root {
+  --text-on-light: var(--miracle-color-dark);
+  --text-on-dark: var(--miracle-color-light);
+  --transition-duration: 0.15s;
+  --border-radius: 64px;
+  --box-shadow: 0 0 48px rgba(34, 40, 49, 0.1);
+  --line-height: 24px;
+  --container-padding: 32px;
+
+  --miracle-shadow-ambient: rgba(34, 40, 49, 0.05) 1px 7px 15px 0px;
```

Comment:
Remove the line-height, container padding, and drop the border radius to `12px` for now. The rest of this file is awesome!